fix(gschema-overrides): Get rid of `files` module dependency

fiftydinar opened the pull request:
Gschema.override files are now placed in /config/gschema-override folder.

fiftydinar committed:
chore(gschema-overrides): Get rid of `files` module dependency

Gschema.override files are now placed in /config/gschema-override folder.

xynydev reviewed (changes requested):
Please change `/config/files/gschema-overrides` to `config/files/gschema-overrides` everywhere. The preceding slash should only be used when an absolute path is in question.

Also the example config in the module.yml needs a docs change.

fiftydinar committed:
chore(gschema-overrides): Don't use 1st slash for new override location in docs

fiftydinar commented:
> Please change `/config/files/gschema-overrides` to `config/files/gschema-overrides` everywhere. The preceding slash should only be used when an absolute path is in question.
> 
> Also the example config in the module.yml needs a docs change.

Fixed

fiftydinar committed:
chore: Use `zz1-` prefix instead of `z1-`

It ensures that no gschema override will bypass user one with override starting with letter z.
